documentation/SupportingDocuments/CAMARA-Release-Workflow-and-Metadata-Proposal.md

```diff
@@ -0,0 +1,272 @@
+# CAMARA Release Workflow and Metadata Specification  
+*Version: Proposal – Concept Phase*
+
+## Summary of Current Situation and Pain Points
+
+CAMARA currently coordinates releases using manual practices across GitHub repositories and external tools (e.g. a Confluence-based tracker), which presents multiple challenges:
+
+- **Manual Release Tracking in Wiki:**  
+  Status tracking is maintained manually using wiki pages, which cannot support automation or flexible reporting. Synchronizing scope and status across repositories requires redundant effort and is error-prone.
+
+- **Informal Release PRs Merged into `main`:**  
+  Releases rely on manually created PRs that are merged into `main` before tagging. These PRs often include version updates, changelog edits, and last-minute changes, introducing risk and inconsistency.
+
+- **Limited CI Enforcement on `main`:**  
+  While MegaLinter is integrated for YAML and Gherkin, there is no structured validation for metadata files or enforcement of CAMARA-specific release guidelines.
+
+- **Unclear Versioning and Branching Strategy:**  
+  Metadata and version fields are frequently changed manually in `main` prior to release, only to be reset afterward. Changes made directly in release PRs are hard to trace and review.
+
+- **Minimal Automation for Changelogs and Checklists:**  
+  CHANGELOG generation is mostly manual. The API Readiness Checklist is not fully automated and depends on manual cross-checks.
+
+## Proposed Objectives and Guiding Principles
+
+We are proposing the following objectives to modernize and automate the CAMARA release process:
+
+- ✅ Clearly separate development intent (planning) from released state (actual).
+- ✅ Keep the `main` branch in a consistent "work in progress" (WIP) state with version fields as `"wip"`.
+- ✅ Use dedicated **release branches per (pre-)release** (e.g. for alpha, rc, release).
+- ✅ Introduce structured **metadata files (in YAML)** to support automation and CI.
+- ✅ Keep CAMARA **release numbering (`rX.Y`)** distinct from API **semantic versioning**.
+- ✅ Enforce CI gating on PRs to `main`.
+- ✅ Restrict release branch changes to authorized release managers.
+- ✅ Automate creation of release branches and generation of release metadata.
+- ✅ Use **Release preparation PRs** into release branches for structured approvals.
+- ✅ Handle specification or implementation feedback via PRs into `main`, and re-update the release branch as necessary.
+
+## Terminology
+
+| Term                   | Description |
+|------------------------|-------------|
+| `meta_release`         | Meta-release timeline (e.g., `Fall26`) |
+| `release_number`       | CAMARA release identifier within (e.g., `r4.1`). Distinct from API SemVer. |
+| `release_status`       | Status of the (planned) (pre-)release: `alpha`, `rc`, or `release`. |
```

Comment:
maybe `public` instead of `release`?

Reply:
Yes, I agree. Done in ee930f5c600b7c122f862292781d47d7d54925f3